doc/index.rst

```diff
@@ -276,18 +276,45 @@ tables. The core annotations of a schema are:
    at a higher level. Typically, the name of the schema can be used as the id.
 :``description``: A textual description of this schema
 :``tables``: the list of tables in the schema. A schema MUST have one or more tables.
+:``version``: Optional schema version description.
 
 Schemas MAY in addition have any number of annotations which provide information about the group of
 tables. Annotations on a group of tables may include:
 
 -  DBMS-specific information for a schema, especially for creating a schema.
 -  IVOA metadata about the table
--  Schema versioning information
 -  Column Groupings
 -  Links to other schemas which may be related
 -  Reference URLs
 -  Provenance
 
+Schema versioning
```

Comment:
Now I'm wondering how much information we really want to duplicate in the sphinx docs and the README, especially given that the readme is in markdown and not rst. Sphinx should be primary, although at the very least we could change to `README.rst` so that at least it would be a simple copy and paste, not reformatting. Something for next time.